packages/kit/src/exports/index.js

```diff
@@ -2,12 +2,27 @@ import { HttpError, Redirect, ActionFailure } from '../runtime/control.js';
 import { BROWSER, DEV } from 'esm-env';
 import { get_route_segments } from '../utils/routing.js';
 
-// For some reason we need to type the params as well here,
-// JSdoc doesn't seem to like @type with function overloads
 /**
- * @type {import('@sveltejs/kit').error}
+ * @overload
```

Comment:
I guess this refers to the `error` method below? It might be clearer to just write it with a single jsdoc where the second param can have multiple types like `{App.Error | {message: string}}`

Reply:
If you've customised `App.Error` then `body` is required; otherwise, it's optional. There's no way to express that without overloads, unfortunately